src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
```

Comment:
You can also try this
`this.health = 100`

this is no need 
`(name, health **= 100**)`

Reply:
agree